### DISCUSSION TITLE
Penn Cove Mussels?

### DISCUSSION THREAD

**Discussion by ChrisMantegna** [RobertsLab/resources#1416] [Ideas]
During science hour we chatted about potentially getting mussels (or making contact) with Ian from the Penn Cove mussel aquaculture outfit. Is that possible? @mattgeorgephd

**Comment by mattgeorgephd** [comment]
Getting mussels won't be an issue - when we are ready we can coordinate a pickup with Tim Jones, the operations manager at Penn Cove.

First step is to find somewhere to put them. Steven is in talks with Jon Wittouck, the Program Operations Specialist in SAFS. Alternatively, I have a chiller/garbage can we can use. They typically hold for 3-weeks in a non-recirc system.

We will also need [algae paste](https://reedmariculture.com/products/shellfish-diet?variant=39282914984041) to feed them. @sr320 do we have a budget to order course supplies?
